teamsyncd/teamsync.cpp

```diff
@@ -102,13 +161,38 @@ void TeamSync::onMsg(int nlmsg_type, struct nl_object *obj)
     if ((nlmsg_type != RTM_NEWLINK) && (nlmsg_type != RTM_DELLINK))
         return;
 
-    string lagName = rtnl_link_get_name(link);
+    /* Introduce check if the interface name is NULL */
+    char *ifName = rtnl_link_get_name(link);
```

Comment:
I do not understand why you change the previous lagName to ifName here. seems ifName is not used anymore afterwards

Reply:
The main reason behind this fix was the observation of a crash in teamsyncd segmentation fault with invalid ifindex and mtu value 

#7  swss::TeamSync::addLag (this=0x3e8, lagName=..., ifindex=-374612960, admin_state=<optimized out>, oper_state=<optimized out>, mtu=3920352592)

So the plan here was to check if the ifName is null … to make sure we don't proceed.